cmd/testscript/README.md

```diff
@@ -0,0 +1,111 @@
+```
+The testscript command runs github.com/rogpeppe/go-internal/testscript scripts
+in a fresh temporary work directory tree.
+
+Usage:
+    testscript [-v] files...
+
+The testscript command is designed to make it easy to create self-contained
+reproductions of command sequences. It is, in many respects, like a poor man's
+Docker.
+
+The format of each script is described in the documentation for
+github.com/rogpeppe/go-internal/testscript.
+
+As a special case, supporting files/directories in the goproxytest subdirectory
+will be served via a github.com/rogpeppe/go-internal/goproxytest server which
+is available to each script by default via the GOPROXY environment variable.
+The server's URL is also available via the GOPROXYTEST environment variable in
```

Comment:
Is this worth doing? If you want to set GOPROXY to something else, you can always set another env var yourself, can't you?

Reply:
Per our discussion, will remove and keep things simple for now.